typescript/src/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "../tsconfig.base.json",
```

Comment:
don't put this in the src folder, just put it before src. No need for `tsconfig.base.json` in that case either

Reply:
I mean aren't the compiled ones going into typescript/dist?

Reply:
yes, but I mean the whole file itself. Not really a comment on a line.
see;
https://github.com/sapphiredev/examples/blob/main/examples/with-typescript-recommended/tsconfig.json